src/index.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "./types";
 
 export { createNodeMiddleware } from "./middleware/node/index";
+export { emitterEventNames } from "./generated/webhook-names";
```

Comment:
Can you add a test to https://github.com/octokit/webhooks.js/blob/master/test/integration/smoke-test.ts? 

I think we should also document the export in the README.

Reply:
sure, will do 😄